python/ml4ir/base/features/feature_fns/categorical.py

```diff
@@ -271,3 +271,101 @@ def categorical_embedding_with_vocabulary_file(feature_tensor, feature_info, fil
     )
 
     return embedding
+
+
+def categorical_indicator_with_vocabulary_file(feature_tensor, feature_info, file_io: FileIO):
```

Comment:
@Ullimague Added another feature function which converts a string categorical variable into one-hot representation. Used this to convert entity labels into their one-hot label representation.

Reply:
Could we add a test for it?